### DISCUSSION TITLE
How to save TPR and FPR

### DISCUSSION THREAD

**Discussion by USTCXSC** [open-edge-platform/anomalib#1593] [Q&A] [answered]
### What is the motivation for this task?

I need to save ROC curves for different models

### Describe the solution you'd like

How can I save the TPR and FPR of the optimal AUROC, or how can I use the saved optimal weights to save TPR and FPR? I need to save ROC curves for different models

**Comment by blaz-r** [comment, marked as answer]
Hello. Check [auroc.py](https://github.com/openvinotoolkit/anomalib/blob/main/src/anomalib/utils/metrics/auroc.py). The file contains code for calculation of tpr and fpr as well as plotting for AUROC. You can modify that or create a separate metric and save it inside `/anomalib/utils/metrics/`.
